exploration/selection-declaration.md

```diff
@@ -0,0 +1,359 @@
+
+Define what effect (if any) the _annotation_ of a _selector_ has on subsequent _placeholders_
+that access the same _variable_.
+
+## Background
+
+_What context is helpful to understand this proposal?_
+
+In MF2, we require that all _selectors_ have an _annotation_.
+The purpose of this requirement is to help ensure that a _selector_ on a given _operand_
+is working with the same value as the _formatter_ eventually used for presentation
+of that _operand_.
+This is needed because the format of a value can have an effect on the grammar used
+in the localized _message_.
+
+For example, in English:
+
+> You have 1 mile to go.
+> You have 1.0 miles to go.
+
+These messages might be written as:
+
+```
+.input {$togo :integer}
+.match {$togo}
+0   {{You have arrived.}}
+one {{You have {$togo} mile to go.}}
+*   {{You have {$togo} miles to go.}}
+
+.input {$togo :number minimumFractionDigits=1}
+.match {$togo}
+0   {{You have arrived.}}
+one {{Unreachable in an English locale.}}
+*   {{You have {$togo} miles to go.}}
+```
+
+It is tempting to want to write these as a shorthand, with the _annotation_ in the _selector_:
+
+```
+.match {$togo :integer}
+0   {{You have arrived.}}
+one {{You have {$togo} mile to go.}}
+*   {{You have {$togo} miles to go.}}
+```
+
+## Use-Cases
+
+_What use-cases do we see? Ideally, quote concrete examples._
+
+1. As a user, I want my formatting to match my selector.
+   This is one of the reasons why MF2 requires that selectors be annotated.
+   When I write a selector, the point is to choose the pattern to use as a template
+   for formatting the value being selected on.
+   Mismatches between these are undesirable.
+
+   ```
+   .match {$num :number minimumFractionDigits=1}
+   one {{This case can never happen in an English locale}}
+   *   {{I expect this formats num with one decimal place: {$num}}}
+   ```
+
+2. As a user, I want to use the least amount of MF special syntax possible.
+3. As a user, I don't want to repeat formatting, particularly in large selection matrices.
+   ```
+   .match {$num1 :integer} {$num2 :number minimumFractionDigits=1}
+   0    0    {{You have {$num1 :integer} ({$num2 :number minimumFractionDigits=1}) wildebeest.}}
+   0    one  {{You have {$num1 :integer} ({$num2 :number minimumFractionDigits=1}) wildebeest.}}
+   0    *    {{You have {$num1 :integer} ({$num2 :number minimumFractionDigits=1}) wildebeest.}}
+   one  0    {{ }}
+   one  one  {{ }}
+   one  *    {{ }}
+   // more cases for other locales that use two/few/many
+   *    0    {{ }}
+   *    one  {{ }}
+   *    *    {{ }}
+   ```
+
+4. As a user (especially as a translator), I don't want to have to modify
+   declarations and selectors to keep them in sync.
+   ```
+   .input {$num :number minimumFractionDigits=1}
+   .match {$num}
+   * {{Shouldn't have to modify the selector}}
+   ```
+   > Note that this is currently provided for by the spec.
+
+5. As a user, I want to write multiple selectors for the same operand.
```

Comment:
```suggestion
5. As a user, I don't want to write multiple selectors for the same operand.
```

Reply:
@gibson042 I don't understand the change proposed? This use case is illustrating a user who _does_ want to select on the same variable using different criteria.